src/citrine/resources/project.py

```diff
@@ -599,9 +602,57 @@ def register(self, name: str, *, description: Optional[str] = None) -> Project:
         project = Project(name, description=description)
         return super().register(project)
 
+    def _list_base(self, *, per_page: int = 1000, archived: Optional[bool] = None):
+        filters = {}
+        if archived is not None:
+            filters["archived"] = str(archived).lower()
```

Comment:
Both `requests` and our testing library just call `str` on a `bool`, leaving it capitalized. And it seems the accounts service doesn't recognize that as a valid boolean.

For the record, I confirmed that at least Orion paths can handle either. So I'm slightly inclined to add a snippet in `Session` that converts any `bool` query parameters to lower-cased strings automatically. But I've left that for the future, when we can more thoroughly ensure there are no ill effects elsewhere in the system.